test/vectors/did/did:example:wallet.json

```diff
@@ -0,0 +1,30 @@
+{
+   "@context":[
+      "https://www.w3.org/ns/did/v1",
+      "https://w3id.org/security/suites/jws-2020/v1"
+   ],
+   "id":"did:example:wallet",
+   "verificationMethod":[
+      {
+         "controller":"did:example:wallet",
+         "id":"did:example:wallet#key1",
+         "publicKeyJwk":{
+            "kty":"EC",
+            "crv":"P-256",
+            "x":"IAY6G0xR4kcVkc_KrIyPb1a50qMCMjHPjVfUunrVGvs",
+            "y":"qzOw6AnovPqnYn2l6MdEBYfVMPNEQTpMxrlcKRDenCk"
+         },
+         "type":"JsonWebKey2020"
+      }
+   ],
+   "service":[
+      {
+         "id":"did:example:wallet#didcomm",
+         "serviceEndpoint":"https://example.com/wallet",
+         "type":"DIDCommMessaging"
+      }
+   ],
+   "keyAgreement":[
```

Comment:
we should add "authentication" per the did spec, because its used to make presentations.